packages/main/src/Carousel.js

```diff
@@ -100,6 +100,18 @@ const metadata = {
 			defaultValue: 0,
 		},
 
+		/**
+		 * Defines when the <code>loadMore</code> event is thrown. If not applied the event will not be thrown.
```

Comment:
I think that this is a little confusing. If the default value says "1" the users would think that the loadMore event will be fired when the navigation is triggered from the item before the last one or from the last one, but not that it won't be fired at all.

If we need a flag to turn on/off this feature we might need a property infinite-scroll  in addition to infinite-scroll-offset :

- infinite-scroll  determines if the loadMore is fired or not at all
- infinite-scroll-offset  determines when to fired it

If you want to keep a single property maybe we should make the default undefined or null, we can discuss it with the others, but "1" to mean the feature is "off" is confusing for sure.